apps/extension/src/utils/taskMasterApi.ts

```
@@ -0,0 +1,1347 @@
+				);
+				return rawResponse;
+			} catch (error) {
+				lastError = error instanceof Error ? error : new Error('Unknown error');
+				console.warn(
+					`TaskMasterApi: Attempt ${attempt}/${this.config.retryAttempts} failed for ${toolName}:`,
+					lastError.message
+				);
+
+				if (attempt < this.config.retryAttempts) {
+					// Exponential backoff
+					const delay = Math.min(1000 * Math.pow(2, attempt - 1), 5000);
```

Biome (1.9.4)

[error] Use the '**' operator instead of 'Math.pow'.

Unsafe fix: Use the '**' operator instead of 'Math.pow'.


(lint/style/useExponentiationOperator)